src/Resources/Tables/Columns/Concerns/InteractsWithResource.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Filament\Resources\Tables\Columns\Concerns;
+
+trait InteractsWithResource
+{
+    public function resourceUrl($resource, $route = 'edit', $shouldOpenInNewTab = false)
+    {
+        $this->url(function ($record) use ($resource, $route) {
```

Comment:
This will need to be wrapped in a `$this->configure()` to ensure that state modifications are made in an orderly way.

Reply:
I don't think it does since `$this->url()` is already calling `$this->configure()`, no? It's purely an alias for another method.

Reply:
Technically you're right, but do you think its worth using `$this->configure()` everywhere to ensure that we're following a pattern? There are some places where we can't avoid using `$this->configure()` unnecessarily due as some parts of the method require it and some already call it in another method.

I'm not sure how big of a consistency issue it is.

Reply:
I don't think it's a huge problem in this instance, because `resourceUrl` is just a proxy alias for the `url` method anyway. It would be different if `resourceUrl` was doing something different and not using `url`, because it would _need_ the configure.